netann/interface.go

```diff
@@ -0,0 +1,23 @@
+package netann
+
+import (
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightningnetwork/lnd/channeldb"
+)
+
+// DB abstracts the required database functionality needed by the
+// ChanStatusManager.
+type DB interface {
```

Comment:
agree that it could be more specific, though given that it's a two line change maybe we can address that when we need to instead of trying to anticipate how the chan status manager's needs are different from some yet-to-be-realized subsystem? :)